docs/source/csv/standard.rst.part

```diff
@@ -1,38 +1,38 @@
-.. _ID:
+.. _Id:
 
-ID
+Id
 ``
-**Data Type:** string
+**Data Type:** `string`_
 
 *This field is required and must be unique.*
 
-Unique identifier for the transaction (i.e. not the voter). IDs should remain consistent, regardless of when an export is created.
+Unique identifier for the transaction (i.e. not the voter). Ids should remain consistent, regardless of when an export is created.
 
 
 
-.. _VoterID:
+.. _VoterId:
 
-VoterID
+VoterId
 ```````
-**Data Type:** string
+**Data Type:** `string`_
 
 *This field is required.*
 
-Identifier that uniquely identifies a voter, but is not required to be unique within the dataset. This number must be obfuscated from the original voter identifier in the jurisdiction's registration database. This field helps analyze how effective each type of transaction is for an individual voter. In addition, if there are questions about a particular voter's transactions, an organization could provide this identifier back to the jurisdiction to glean more information about the voter's experience.
+Identifier that uniquely identifies a voter but is not required to be unique within the dataset. This number must be obfuscated from the original voter identifier in the jurisdiction's registration database. This field helps analyze how effective each type of transaction is for an individual voter.
 
 
 
-.. _ApplicationRequestMethod:
+.. _RequestMethod:
 
-ApplicationRequestMethod
-````````````````````````
-**Data Type:** string
+RequestMethod
+`````````````
+**Data Type:** `string`_
 
 *This field is required.*
 
-Specifies the method by which the application request was received. Possible values are **email**, which signifies the application was received via email; **fax**, which denotes the application was received by fax; **in-person**, which means the application was delivered by a person (NB: this does not necessarily signify the voter themselves delivered the application); **mail**, which signifies the application was received via mail; **online**, which means the application was submitted via online form or system; **phone**, which means the request was placed via phone call; **untracked**, which signifies that this variable is not tracked in the system; and, **other**, which is a catch-all for any values that fall outside of the other values.
+Specifies the method by which the UOCAVA request was triggered. Possible values are **email**; **fax**; **in-person**, which means the request was triggered by a person; **mail**; **online**, which means the request was submitted via online form or system — through the elections office or other registration agency; **phone**, which means the request was placed via phone call; **untracked**, which signifies that this variable is not tracked in the system; and, **other**, which is a catch-all for any values that fall outside of the other values.
```

Comment:
I don't have a strong opinion either way. I think election offices understand in-person means "a person" not necessarily the voter. I removed this because it was removed in the O365 doc.

Reply:
Sounds good. Since I just noticed it, maybe also remove UOCAVA here as well?